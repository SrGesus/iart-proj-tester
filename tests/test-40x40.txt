VC	LH	LV	LH	VC	VC	BB	FC	VB	LH	LH	BD	LV	LH	VB	VC	LV	VC	VB	BD	FE	VE	LV	LV	BC	LV	VD	FC	LV	BB	LV	FD	VB	BE	VE	VD	BC	VB	VB	VD
LH	FB	VB	LV	VE	LH	LH	VB	VC	FB	VB	VC	VC	VC	VB	VE	VE	VB	LV	VE	LV	BD	FC	VD	VD	FE	VD	VB	VE	VD	VB	LH	VD	LV	LH	LV	LH	VE	VC	FC
LH	LV	VD	VD	FE	BB	LV	VC	LV	BB	LH	FD	LV	BE	FB	VD	VB	FE	LV	FE	VD	LV	LV	VB	VE	BB	LV	VB	VC	LV	VE	VC	LV	VE	LH	FB	VC	BD	LV	VB
LH	BD	LV	VB	VE	BC	FD	VE	LV	VE	LV	BC	VB	VB	LV	BE	VD	BD	VB	BE	VC	LH	VD	VC	VC	VE	LV	LH	LV	VC	VD	VE	VE	LH	VE	VE	VC	VB	VE	LV
LV	LH	VB	VC	LV	LH	VD	VD	VC	LV	VE	LH	VC	BB	LV	VC	LH	LV	VC	VE	VB	FB	LH	VD	VD	LV	VC	BD	FD	VB	VB	FB	LV	VB	LH	VC	LV	FE	BE	LH
LV	VB	VE	LV	LH	LV	LH	FB	LV	VD	FD	LV	LV	FB	VE	LH	VD	LV	LH	VE	VD	VC	VB	VD	BC	VB	FD	LV	VD	LH	LV	BE	VD	BC	LV	VB	VB	VD	FD	LV
VE	VD	FC	BD	VD	LV	LV	LV	LH	VE	BC	VC	VC	VE	LV	FE	VB	BD	VD	BC	VD	VC	FC	VC	VE	LV	VE	VD	BE	LH	LH	LV	VB	VE	FC	LH	FE	VD	LH	BD
VC	BE	VB	LH	VE	LH	LH	LV	VC	VC	VD	LV	FE	VD	BB	VE	LV	LH	FB	LH	VB	LH	LV	VB	VC	VC	VC	LH	LH	LV	LV	VD	VC	LV	LH	VC	VB	LV	VB	LH
LV	FD	VB	VE	LH	FB	BE	BD	FC	VD	VB	VE	VD	FE	LV	LV	VB	LH	LH	VE	VB	FE	VD	BB	VB	VE	VB	VC	LV	LH	FD	LH	FB	BD	LH	VB	VD	LH	VE	LH
BD	LH	LH	VC	VD	LH	VB	VB	LV	LV	VB	FD	LV	FC	LV	BC	FD	LV	BB	LH	BE	LH	BD	LV	FE	VD	VD	VB	VE	VB	LH	VB	VD	VD	VB	VD	LV	VB	LH	VC
VE	LV	LH	LH	FE	VB	BB	VD	VC	LV	FE	VD	VB	FB	VD	BD	LH	BE	VB	VD	VC	VE	VB	VE	VB	LV	VD	VC	LH	LV	VC	BD	VE	VE	BC	LV	VB	LH	VB	VB
VD	LV	LH	LV	VD	LH	FE	VD	VC	VE	VD	VE	LH	VD	LH	FC	VC	LH	LH	VB	LH	VE	VB	FE	VD	VB	LV	VB	VD	LV	LH	VD	VD	FD	VB	LH	LH	VE	FE	LH
BE	VB	FC	LV	BE	VB	VB	VB	VD	BD	VC	LH	VC	LV	VE	LH	BD	FC	VC	VB	BD	FC	VD	BB	VE	FE	BE	VE	LH	VD	BC	FC	VC	LV	BE	VD	VC	LH	VC	LV
LH	VD	VE	VC	LV	VE	LV	VD	VB	VE	BD	VB	LV	BC	LH	FC	VC	LV	VB	VB	BD	VB	BD	FB	LH	VD	VC	VB	LV	VB	FE	VC	LH	LH	VC	VE	VE	FD	BE	BC
LH	FB	VD	VE	FB	VE	VE	FE	VC	VC	FC	LH	VE	VB	VC	VB	LV	FD	VB	LV	VB	LV	VB	LV	VB	VE	LH	LH	VC	VB	LV	BC	VC	FC	VE	BC	LV	VC	FE	VD
LV	VE	BC	LV	BD	VD	LV	VD	LV	VE	VD	LH	VE	VB	LV	VE	VD	LV	VB	LH	VC	VB	LV	VD	VE	BE	LV	VD	LH	VB	VC	VE	VC	VE	VD	VD	FD	LH	VC	VD
VB	VC	LH	VD	VC	LH	FE	VB	VE	FE	LV	BD	VB	LH	LV	VC	VC	BC	BE	FB	LH	VC	FB	LV	FD	VB	VD	LH	LH	LV	LH	VC	LV	VD	VD	LH	VC	VB	VD	LH
VE	BD	VD	VB	LH	VD	BE	BC	FE	BE	VB	FB	VE	BC	LV	FE	BB	VB	VC	LH	VB	LH	VE	VE	VE	VC	LH	VC	BD	LV	VC	VE	FE	LV	LV	BC	VD	VE	LH	BC
VC	VD	VD	VC	FD	BB	BC	FC	VB	BB	LH	VD	LV	LV	VC	VE	VC	VC	FD	LH	BB	BD	VD	VB	LV	VC	BC	FE	VE	LH	VC	LV	VB	VE	VC	VD	LV	BB	FE	VD
FC	LV	LH	LH	VC	VD	FB	VD	VD	VC	VB	VD	VD	VE	FB	BE	VD	VC	VD	VB	VD	BE	FE	VC	VD	VB	VC	VC	BE	VE	LH	FC	LV	LV	VD	LH	VC	FC	VE	VD
BC	VE	LV	LV	VE	LV	LH	BE	VE	VB	VD	LH	LH	LV	VB	FE	VB	VD	VC	VB	VE	VD	VD	LV	VD	LV	FE	LV	FB	VE	BB	VC	LH	BC	LH	FB	VD	LV	VE	LV
VB	VD	LH	VB	LH	LH	VD	FE	LV	VD	LH	BD	LV	FC	VB	LV	LV	VD	VD	LH	VB	FE	VB	VD	LH	BC	VC	LH	VC	VD	VB	VE	BB	VC	VC	LH	VE	VD	VC	LV
VD	VD	LH	FB	LV	LV	BB	LH	VC	BB	VC	LV	VE	BD	LV	FD	VC	LV	VD	VB	LH	VC	FE	LH	VE	VE	VD	BB	LH	FB	LH	FC	VD	FC	LV	VD	VD	LH	VB	VC
VC	FD	LV	VE	LV	VD	VD	VC	VB	VC	LH	FC	LH	VD	LH	LV	VE	VC	LH	BC	VC	VB	VC	BD	FC	VC	VE	LV	LH	LV	VC	LH	LH	VB	LV	VB	VE	LV	VB	VB
VC	BD	VB	BD	FE	LH	LH	FE	VD	VE	VE	LH	VC	FD	BB	LH	VB	LH	VB	FB	VE	BD	FE	VD	VB	VB	VE	VB	LH	VB	LH	BC	VB	LH	LH	VB	VD	LV	LV	FE
LH	LV	VC	VE	VD	VD	BD	LV	LH	VB	FB	BC	VD	VB	VB	FB	FD	LH	VC	VE	FE	LH	FE	VE	VE	VE	LV	FD	LV	VC	LH	VB	LV	VD	VB	LH	VD	VD	BD	VC
LH	FD	VD	VD	LH	FC	VD	VC	FD	LV	BD	VD	VE	VB	FE	BC	BD	BE	FC	LH	VE	BD	BE	BB	FE	LH	VE	BC	VE	VB	FD	VD	VD	VE	VD	VB	VC	VD	VB	LV
BE	LH	VE	LH	LH	VE	LV	BB	VC	VE	BE	VB	VB	VE	LH	LV	FC	VC	VE	VE	VD	FE	FE	VD	VE	LV	FB	LH	FC	VB	BE	VB	FC	LV	BB	FC	VC	VE	VC	VC
VD	VE	VB	VE	VB	LH	VB	VB	VC	LH	FC	LH	LH	VB	VE	VC	VC	LH	BB	VE	LV	LV	VC	VD	LH	LH	LH	VE	VE	LV	LV	VD	VC	LH	LH	VD	BC	FC	VC	VB
FC	VC	VC	FB	VE	VD	LH	FE	LV	BC	LV	VB	VB	VB	VB	VB	VB	VB	LV	FD	BE	BE	LH	FD	VC	BD	VB	LV	BC	VC	LH	BE	LV	VC	LH	LH	VE	LH	LH	BD
BE	VD	LH	VE	BE	LH	VE	LH	LV	LV	LH	BB	VE	VE	VC	LH	FD	VD	LH	VE	VC	BC	VE	VE	BC	VB	VB	LH	VB	VC	VC	VD	VB	FD	LV	LV	VC	VB	FB	VC
LH	LH	VE	VD	FC	VD	LH	LH	BD	VD	FC	LH	FD	VC	LH	VC	VD	VD	VB	VD	FB	LV	VB	VB	VC	FE	LV	FC	VE	BE	BE	VB	LV	LH	VE	VC	FD	VE	LH	VE
LV	VD	VD	VC	LV	LH	LH	LV	VB	LH	LV	VB	VB	VC	LH	VB	BB	LH	LH	BD	VC	LV	LH	LH	VE	LH	VB	VD	BE	VC	LH	LH	LV	VC	BC	BD	BE	VE	VD	LV
BE	FE	VD	LV	BC	BE	LH	FD	VC	BD	BC	FD	BB	VB	VB	VC	VC	LV	FE	VC	VB	LV	LH	VC	VD	VC	BE	LV	VE	VD	BD	FD	VD	VB	FC	LV	LH	LV	VC	BC
FB	VC	BB	LH	VC	FB	VB	LH	BC	FC	VE	LV	BC	FE	VE	LV	LH	LV	LH	LV	BC	VC	VE	FB	LH	BE	VE	LV	VB	FB	LV	VB	LH	BD	LH	BD	FE	VC	VD	FC
VC	VC	LH	VE	LH	LH	BC	FE	VC	VC	LV	FC	VE	VE	LH	BC	LV	VB	VB	VC	LH	VC	VD	VB	VC	VD	LV	VD	LV	VD	VE	LV	VD	LH	VB	LH	VB	VC	VB	VD
VD	VD	LH	LH	VE	LH	VE	VB	VB	BE	VD	VE	LV	VE	FD	VE	VE	VE	VE	LV	LH	LV	VC	VC	VD	LV	VE	FE	VE	BE	LV	VC	LH	FC	LV	LV	LH	VC	LH	BB
VB	VB	FB	LV	LV	VE	VD	VC	VE	LH	VE	VB	VD	VE	BB	VC	LH	FE	LH	VC	LV	VD	VE	VE	BB	FB	LH	VC	VC	VC	VD	VE	VE	BB	VD	FB	VB	VB	VC	VB
BE	VE	VD	VE	FB	VC	LH	VE	LV	FE	VE	LH	VD	VE	VD	LH	LV	VB	LV	VB	LV	VB	VC	LV	LH	VE	VD	LH	VD	VC	FD	LV	VE	VD	VD	VC	VB	VE	LH	FE
FB	VC	BE	LH	LH	LH	LV	VB	VE	LH	LV	LV	LH	LH	LH	VB	VC	BC	FB	VB	BD	VD	FE	VE	VD	BC	LV	VD	FB	BB	LH	VC	VC	LV	VB	VD	VB	FB	BE	VE
