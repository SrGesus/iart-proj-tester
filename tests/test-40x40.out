VB	LH	LH	LH	VE	VB	BB	FE	VB	LH	LH	BB	LH	LH	VE	VB	LH	VE	VB	BB	FE	VB	LH	LH	BB	LH	VE	FD	LH	BB	LH	FE	VB	BB	VE	VB	BB	VE	VB	VE
LV	FB	VB	LH	VC	LV	LV	VB	VC	FB	VB	VC	VB	VE	VD	VC	VB	VC	LV	VD	LH	BC	FE	VB	VC	FB	VD	VE	VB	VC	VB	LH	VC	LV	LV	LV	LV	VD	VC	FC
LV	LV	VD	VE	FD	BE	LV	VD	LH	BE	LV	FB	LV	BD	FE	VB	VC	FB	LV	FB	VB	LH	LH	VC	VB	BC	LH	VC	VD	LH	VC	VB	LH	VC	LV	FC	VD	BB	LH	VE
LV	BD	LH	VC	VB	BE	FC	VB	LH	VC	LV	BD	VC	VD	LH	BE	VB	BE	VD	BE	VD	LH	VE	VB	VC	VB	LH	LH	LH	VE	VB	VC	VB	LH	VC	VB	VE	VD	VE	LV
LV	LV	VB	VE	LV	LV	VB	VC	VB	LH	VC	LV	VB	BB	LH	VC	LV	LV	VB	VC	VB	FE	LV	VD	VE	LV	VB	BB	FE	VD	VC	FB	LV	VB	LH	VC	LV	FD	BE	LV
LV	VD	VC	LV	LV	LV	LV	FB	LV	VB	FE	LV	LV	FC	VB	LH	VC	LV	LV	VB	VC	VB	VC	VB	BC	VC	FC	LV	VB	LH	LH	BC	VC	BD	LH	VE	VD	VE	FC	LV
VD	VE	FD	BC	VC	LV	LV	LV	LV	VD	BB	VC	VD	VE	LV	FB	VB	BE	VD	BE	VB	VC	FD	VC	VB	LH	VE	VD	BC	LH	LH	LH	VE	VD	FE	LV	FB	VD	LH	BE
VB	BE	VB	LH	VE	LV	LV	LV	VD	VE	VD	LH	FE	VD	BC	VC	LV	LV	FB	LV	VD	LH	LH	VE	VD	VE	VD	LH	LH	LH	LH	VE	VD	LH	LH	VC	VD	LH	VE	LV
LV	FC	VD	VE	LV	FC	BD	BC	FE	VD	VE	VB	VE	FD	LH	LH	VC	LV	LV	VD	VE	FD	VE	BD	VE	VD	VE	VB	LH	LH	FE	LV	FD	BB	LH	VE	VB	LH	VC	LV
BD	LH	LH	VC	VD	LH	VC	VB	LH	LH	VC	FC	LV	FD	LH	BB	FE	LV	BD	LH	BC	LH	BE	LV	FC	VB	VC	VD	VE	VB	LH	VC	VB	VC	VB	VC	LV	VB	LH	VC
VD	LH	LH	LH	FE	VB	BB	VC	VB	LH	FE	VB	VC	FD	VE	BD	LH	BC	VC	VB	VE	VB	VC	VD	VE	LV	VB	VE	LV	LV	VB	BB	VC	VB	BC	LH	VC	LV	VB	VE
VB	LH	LH	LH	VE	LV	FC	VB	VC	VB	VE	VD	LH	VE	LV	FC	VB	LH	LH	VC	LV	VD	VE	FB	VD	VC	LV	VD	VC	LV	LV	VD	VE	FC	VB	LH	LH	VC	FC	LV
BD	VE	FD	LH	BC	VC	VB	VC	VB	BE	VD	LH	VE	LV	VD	LH	BC	FE	VB	VE	BD	FE	VD	BC	VE	FD	BE	VB	LH	VC	BD	FE	VD	LH	BC	VE	VB	LH	VE	LV
LV	VD	VE	VB	LH	VE	LV	VB	VC	VD	BB	VE	LV	BD	LH	FE	VB	LH	VC	VD	BE	VB	BB	FE	LV	VB	VC	VD	LH	VE	FC	VB	LH	LH	VE	VD	VC	FD	BC	BE
LV	FB	VD	VC	FB	VD	VC	FC	VB	VE	FC	LV	VD	VC	VB	VE	LV	FB	VB	LH	VC	LV	VD	LH	VC	VD	LH	LH	VE	VD	LH	BE	VB	FE	VD	BB	LH	VE	FD	VC
LV	VD	BB	LH	BE	VB	LH	VE	LV	VD	VE	LV	VB	VE	LV	VD	VC	LV	VD	LH	VE	VD	LH	VE	VB	BB	LH	VE	LV	VB	VE	VD	VC	VB	VE	VD	FE	LV	VB	VE
VD	VE	LV	VB	VC	LV	FB	VD	VC	FB	LV	BD	VC	LV	LV	VB	VE	BD	BB	FE	LV	VB	FE	LV	FC	VD	VE	LV	LV	LV	LV	VB	LH	VC	VD	LH	VE	VD	VC	LV
VB	BC	VC	VD	LH	VC	BD	BB	FE	BD	VC	FC	VB	BE	LV	FC	BD	VC	VD	LH	VC	LV	VB	VC	VB	VE	LV	VD	BE	LV	VD	VC	FD	LH	LH	BB	VC	VB	LH	BE
VD	VE	VB	VE	FD	BB	BE	FC	VB	BC	LH	VE	LV	LV	VD	VE	VD	VE	FD	LH	BB	BE	VD	VE	LV	VD	BE	FD	VC	LV	VB	LH	VE	VB	VE	VD	LH	BE	FD	VC
FB	LV	LV	LV	VB	VC	FC	VB	VC	VB	VE	VD	VC	VD	FE	BD	VE	VD	VE	VB	VC	BD	FE	VD	VC	VB	VC	VB	BB	VC	LV	FB	LV	LV	VD	LH	VE	FC	VB	VE
BD	VC	LV	LV	VD	LH	LH	BE	VB	VC	VD	LH	LH	LH	VE	FC	VD	VE	VD	VC	VB	VC	VB	LH	VE	LV	FB	LV	FC	VB	BC	VC	LV	BD	LH	FE	VD	LH	VC	LV
VD	VE	LV	VD	LH	LH	VE	FC	LV	VB	LH	BB	LH	FE	VD	LH	LH	VC	VB	LH	VC	FB	VD	VE	LV	BD	VC	LV	VB	VC	VB	VE	BD	VC	VB	LH	VE	VB	VE	LV
VB	VC	LV	FD	LH	LH	BC	LH	VC	BD	VE	LV	VB	BB	LH	FE	VB	LH	VC	VB	LH	VC	FB	LV	VD	VC	VB	BE	LV	FB	LV	FC	VD	FE	LV	VB	VC	LV	VD	VC
VD	FE	LV	VB	LH	VE	VB	VE	VB	VC	LV	FC	LV	VD	LH	LH	VC	VB	LH	BC	VE	VB	VC	BD	FE	VB	VC	LV	LV	LV	VD	LH	LH	VE	LV	VD	VE	LV	VB	VE
VB	BB	VC	BD	FE	LV	LV	FC	VD	VE	VD	LH	VC	FD	BB	LH	VE	LV	VB	FE	VD	BE	FD	VC	VB	VC	VB	VC	LV	VD	LH	BB	VE	LV	LV	VB	VC	LV	LV	FC
LV	LV	VB	VC	VB	VC	BD	LH	LH	VC	FD	BB	VE	VB	VC	FB	FC	LV	VD	VE	FB	LV	FD	VE	VD	VE	LV	FB	LV	VB	LH	VC	LV	VD	VC	LV	VB	VC	BD	VE
LV	FC	VD	VE	LV	FB	VD	VE	FD	LH	BB	VC	VD	VC	FB	BD	BB	BC	FE	LV	VD	BC	BB	BC	FE	LV	VD	BE	VD	VC	FB	VB	VC	VB	VE	VD	VC	VB	VC	LV
BD	LH	VE	LV	LV	VD	LH	BE	VB	VE	BD	VE	VB	VE	LV	LV	FC	VB	VE	VD	VE	FB	FC	VB	VE	LV	FB	LV	FD	VE	BD	VC	FB	LV	BD	FE	VB	VC	VB	VC
VD	VE	VD	VC	VD	LH	VE	VD	VC	LV	FC	LV	LV	VD	VC	VD	VE	LV	BD	VE	LV	LV	VB	VC	LV	LV	LV	VD	VE	LV	LV	VB	VC	LV	LV	VB	BC	FE	VD	VE
FB	VD	VE	FB	VB	VE	LV	FD	LH	BC	LH	VC	VD	VE	VB	VE	VD	VC	LV	FC	BD	BE	LV	FB	VD	BE	VD	LH	BC	VC	LV	BD	LH	VC	LV	LV	VB	LH	LH	BE
BD	VE	LV	VD	BE	LV	VD	LH	LH	LH	LH	BB	VE	VD	VC	LV	FD	VE	LV	VB	VC	BD	VC	VD	BB	VC	VB	LH	VE	VB	VC	VD	VE	FB	LV	LV	VD	VE	FD	VC
LV	LV	VD	VE	FC	VD	LH	LH	BB	VE	FB	LV	FC	VB	LH	VC	VB	VC	VD	VC	FB	LV	VB	VE	VD	FE	LV	FB	VD	BC	BB	VE	LV	LV	VD	VC	FB	VD	LH	VE
LV	VD	VE	VD	LH	LH	LH	LH	VC	LV	LV	VD	VE	VD	LH	VE	BD	LH	LH	BB	VC	LV	LV	LV	VB	LH	VC	VD	BB	VE	LV	LV	LV	VD	BB	BB	BE	VB	VE	LV
BD	FE	VD	LH	BB	BB	LH	FE	VB	BC	BC	FE	BD	VE	VB	VC	VD	LH	FE	VD	VE	LV	LV	VD	VC	VB	BB	LH	VC	VD	BE	FC	VD	VE	FC	LV	LV	LV	VD	BE
FC	VB	BB	LH	VC	FC	VB	LH	BC	FE	VB	LH	BE	FC	VD	LH	LH	LH	LH	LH	BE	VD	VC	FD	LH	BE	VD	LH	VE	FB	LV	VB	LH	BC	LH	BE	FC	VD	VE	FC
VB	VC	LV	VB	LH	LH	BC	FE	VB	VE	LV	FD	VC	VB	LH	BB	LH	VE	VB	VE	LV	VB	VE	VB	VE	VD	LH	VE	LV	VD	VC	LV	VB	LH	VE	LV	VB	VE	VD	VE
VD	VE	LV	LV	VB	LH	VE	VB	VC	BD	VC	VB	LH	VC	FB	VD	VE	VD	VC	LV	LV	LV	VD	VC	VD	LH	VE	FC	VD	BB	LH	VC	LV	FB	LV	LV	LV	VD	LH	BE
VB	VC	FC	LV	LV	VB	VC	VD	VE	LV	VB	VC	VB	VE	BD	VE	LV	FD	LH	VC	LV	VD	VE	VB	BB	FE	LV	VB	VE	VD	VE	VB	VC	BD	VC	FC	VD	VE	VB	VC
BD	VE	VB	VC	FC	VD	LH	VE	LV	FC	VD	LH	VC	VD	VC	LV	LV	VB	LH	VE	LV	VB	VC	LV	LV	VB	VC	LV	VD	VE	FC	LV	VB	VC	VB	VE	VB	VC	LV	FB
FC	VD	BC	LH	LH	LH	LH	VC	VD	LH	LH	LH	LH	LH	LH	VC	VD	BC	FE	VD	BC	VC	FD	VC	VD	BC	LH	VC	FD	BC	LH	VC	VD	LH	VC	VD	VC	FD	BC	VC
